tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
```

Comment:
Most of the users don't know what magna is, Use general description for URL

Reply:
Thanks @vamahaja
Removed "magna" from doc string and added "url" for general description.